presto-parser/src/test/java/com/facebook/presto/sql/parser/TestSqlParser.java

```diff
@@ -750,6 +745,11 @@ public void testDecimal()
         assertExpression("DECIMAL '-12'", new DecimalLiteral("-12"));
         assertExpression("DECIMAL '+.34'", new DecimalLiteral("+.34"));
         assertExpression("DECIMAL '-.34'", new DecimalLiteral("-.34"));
+
+        assertExpression("123.", new DecimalLiteral("123."));
```

Comment:
I don't think this is a valid literal in SQL, and that it's a bug for Presto to accept them.